data/index.js

```diff
@@ -89,6 +89,18 @@ export function registerReducer( reducerKey, reducer ) {
 	return store;
 }
 
+/**
+ * The combineReducers helper function turns an object whose values are different
+ * reducing functions into a single reducing function you can pass to registerReducer.
+ *
+ * @param {Object} reducers	An object whose values correspond to different reducing
+ * 							functions that need to be combined into one.
```

Comment:
Tabbing is off here. Align with whitespace.

Reply:
Prettier, where are you? 😃